src/Hangman/Domain/Updaters/LetterGuessed.updater.ts

```diff
@@ -0,0 +1,26 @@
+import { Game as GameProjection } from '../../ReadModels/game.entity';
+
+import {
+  IViewUpdater,
+  ViewUpdaterHandler,
+} from '@berniemac/event-sourcing-nestjs';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { LetterGuessedEvent } from '../Events/LetterGuessed.event';
+
+@ViewUpdaterHandler(LetterGuessedEvent)
+export class LetterGuessedUpdater implements IViewUpdater<LetterGuessedEvent> {
+  constructor(
+    @InjectRepository(GameProjection)
+    private gamesProjectionRepository: Repository<GameProjection>,
+  ) {}
+
+  async handle(event: LetterGuessedEvent) {
+    this.gamesProjectionRepository.update(
+      {
+        gameId: event.id,
+      },
+      { lettersGuessed: event.lettersGuessed },
```

Comment:
this is not a nice solution, now I have to pass the state of the aggregate in the event, in stead of only the change. 
But how to update the view if I don't have the state of the aggregate ? 

Use the state of the view, add the new letter and put it back?